verticapy/vdataframe.py

```diff
@@ -7896,8 +7896,27 @@ def join(
     ----------
     input_relation: str/vDataFrame
         Relation to use to do the merging.
-    on: dict, optional
-        Dictionary of all different keys. The dict must be similar to the following:
+    on: dict / list, optional
+        List of 3-tuples. Each tuple must include (key1, key2, operator) where
```

Comment:
maybe write: If it is a list then....
and below we can write : if it a dict then

Maybe use two indents for each to make it more clear.

Reply:
ok